.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build-deeploy:
     runs-on: ubuntu-22.04
     container:
-      image: ghcr.io/pulp-platform/deeploy:main
+      image: ghcr.io/tahaelbayad/deeploy:main
```

Comment:
Exactly. First, we check that the modified CI passe uses the modified Docker Container. Then, we reverted the container link to the point at the main repo container. Finally, we merge and manually trigger the docker build with the new Dockerfile, then restart the CI on the main repo that would use the new docker.

Reply:
NIT, but then I suggest adding a todo in the description (using the "- [ ]" notation. This will also ensure that the PR is not merged before all open tasks are completed.